python/ray/train/_internal/session.py

```diff
@@ -325,6 +319,32 @@ def get_next(self) -> Optional[_TrainingResult]:
         # Return None if there are no more results to fetch.
         return result
 
+    def _get_or_create_inter_actor_queue(self):
+        """Get or create the inter-actor queue."""
+        if self._inter_actor_queue is None:
+            self._inter_actor_queue = ray_queue.Queue(1)
```

Comment:
we should still require 0 cpus here so user code doesn't hang forever

```suggestion
            self._inter_actor_queue = ray_queue.Queue(1, actor_options={"num_cpus": 0})
```

Reply:
done.